README.md

```diff
@@ -107,7 +107,7 @@ Run recording daemon:
 
 ```shell
 $ sudo cp ~/.cargo/bin/below /bin/below  # if using cargo-install
-$ sudo cp resctl/below/etc/below.service /etc/systemd/system
+$ sudo cp etc/below.service /etc/systemd/system  # from below directory
```

Comment:
```suggestion
$ sudo cp etc/below.service /etc/systemd/system  # from below directory
```
Recommend dropping `# from below directory`. Wasn't immediately clear what this meant and I think the command is obvious enough